catroid/src/main/java/org/catrobat/catroid/formulaeditor/common/Conversions.kt

```diff
@@ -32,6 +32,9 @@ object Conversions {
     private val colorPattern = Pattern.compile("#\\p{XDigit}{6}")
 
     private fun tryParseDouble(argument: String): Double? {
+        if (argument.contains('f') || argument.contains('d')) {
+            return null
+        }
```

Comment:
please replace the if with 
`if (argument.matches(Regex("[fFdD]\$"))){`
as currently the usecase ("F") which promptet this task is ignored